WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -34,10 +33,10 @@ public Arena(TimeSpan period, Network network, WabiSabiConfig config, IRPCClient
 
 		public HashSet<Round> Rounds { get; } = new();
 		private AsyncLock AsyncLock { get; } = new();
-		public Network Network { get; }
+		private Network Network { get; }
 		public WabiSabiConfig Config { get; }
 		public IRPCClient Rpc { get; }
-		public Prison Prison { get; }
+		private Prison Prison { get; }
```

Comment:
This is really what is important about this PR. `Prison` should not be accessible from outside, except in tests.